include/backup_restore.hpp

```diff
@@ -78,15 +77,15 @@ class BackupRestore
                                 const std::string& i_srcPath,
                                 const std::string& i_dstPath);
 
-    // Parsed system JSON config file.
+    // System JSON config JSON object.
     nlohmann::json m_sysCfgJsonObj{};
 
-    // Parsed backup and restore's JSON config file.
-    nlohmann::json m_backupRestoreCfgJsonObj{};
+    // Backup and restore config JSON object.
+    nlohmann::json m_backupAndRestoreCfgJsonObj{};
 
     // Indicates if backup and restore has been performed.
     static bool m_backupAndRestoreDone;
```

Comment:
m_isBackupAndRestoreCalled or m_isBackupAndRestoreTriggered 
which is better or should we retain as it is?

Reply:
We will fix this in the commit which brings in an enum to handle the states.
There we need to rename this as m_isBackupAndRestoreStatus